core/controllers/domain_objects_validator.py

```diff
@@ -36,3 +39,50 @@ def validate_exploration_change(obj):
     # No explicit call to validate_dict method is necessary, because
     # ExplorationChange calls it while initialization.
     exp_domain.ExplorationChange(obj) # type: ignore[no-untyped-call]
+
+
+def validate_new_config_property_values(obj):
+    # type: (Dict[Any, Any]) -> None
```

Comment:
Since we know that the name is a string, please add a stricter check.

Reply:
Done.